lib/nexmo/oas/renderer/views/open_api/_parameters.erb

```diff
@@ -59,11 +59,11 @@
           <span class="constraint">Required</span>
         <% end %>
 
-        <% if parameter.minimum %>
-          | <span class="constraint">Min:</span> <%= parameter.minimum %>
+        <% if parameter.minimum || parameter.raw['minLength'] %>
```

Comment:
When this displays the maximum length, will it say `Max`, or will it display `Max Length`?

also - should we consider incorporating `minItems` and `maxItems`?

Reply:
It'll say `Max`. The length is implicit for strings (though if we get feedback that it's misleading I'm open to changing it too)

Reply:
Sounds good